samples_project/Assets/SampleViewer/Samples/FlightSim/FlightSimSample/FlightController_NewInput.cs

```diff
@@ -0,0 +1,153 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+using System;
+public class FlightController_NewInput : MonoBehaviour
+{
+    [Header ("Private Variables")]
+    private bool isGrounded = false;
+    private bool speedIncreasing;
+    private float rotationX;
+    private float rotationY;
+    private float rotationZ;
```

Comment:
Can we not just use the location component rotational properties?

Reply:
The way I am doing it won't work with the rotation properties on the location component. It won't let me set them to variables in order to modify them. In order to modify them on the normal transform, each angle has to be either the x, y, z on a vector 3. I tried a ton of ways and ultimately this was what worked the best.